### PR TITLE
sztp: add device boostraping os before sztp

### PR DESCRIPTION
**Pull request by glimchb**
- sztp: add comment about ignoring dhcp options
- sztp: add device boostraping os before sztp

### CODE REVIEW

**Review by mestery — APPROVED** [review]
Well thought out and makes sense.

File: ZTP.md [hunk 1]
```diff
@@ -88,6 +97,8 @@ Note: DHCP is completely *untrusted* in this flow, we are not going to secure th
 
 Note: DHCP is *not* the only option for discovery. mDNS, SLAAC and others can also be supported as part of sZTP by some devices.
 
+Note: When sZTP is enabled, other DHCP options (except defined in the RFC, code 136,143) are *ignored*.
```
Comment (shachartal) [inline ZTP.md:100]: well, you still want to configure the IP stack for this client (so you need to consume netmask and router list options at least).

**Review by shachartal — COMMENTED** [review]
small oversight around DHCP response options

File: ZTP.md (hunk 1 above)
Comment (glimchb) [reply ZTP.md:100]: correct, I mean ZTP options like 66 & 67...

**Comment by timmichels** [comment]
Thanks Boris,  Excellent work using IETF and IEEE methods to fill out an important workflow.

**Pull request merged**